ab_testing_tool/settings/test_settings.py

```diff
@@ -7,3 +7,12 @@
 if not ENV_SETTINGS.get('enable_test_logging'):
     for logger in LOGGING['loggers']:
         LOGGING['loggers'][logger]['handlers'] = ['null']
+
+# make tests faster
+DATABASES = {
```

Comment:
I'm on board with running the tests on sqlite for the precommit smoke testing, but unless it's completely not an option (eg. oracle), we should be running other tests on the same type of db we're running in production.